scipy/misc/pilutil.py

```diff
@@ -94,10 +94,10 @@ def bytescale(data, cmin=None, cmax=None, high=255, low=0):
         cscale = 1
 
     scale = float(high - low) / cscale
-    bytedata = (data * 1.0 - cmin) * scale + 0.4999
+    bytedata = (data * 1.0 - cmin) * scale + low
     bytedata[bytedata > high] = high
-    bytedata[bytedata < 0] = 0
-    return cast[uint8](bytedata) + cast[uint8](low)
+    bytedata[bytedata < low] = low
+    return cast[uint8](bytedata.round())
```

Comment:
That's a tricky one. I'd tend to say that the 0.4999 is incorrect. However @WarrenWeckesser wanted a backwards compatible change (https://github.com/scipy/scipy/issues/5305#issuecomment-145130530).